### ISSUE TITLE
Direct links to Discord voice channels

### ISSUE THREAD

**Issue by erwa** [cardinalitypuzzles/smallboard#458]
I forget if this was already discussed, but it would be nice if smallboard had direct links to Discord voice channels in addition to the text channels.

**Comment by rgossiaux** [comment]
Yeah I think we wanted to have that but Discord doesn't really support it. I think it is possible to programmatically move someone from one voice channel to another, which we could use to have a button on the site that joined the voice channel for you (assuming you're in a voice chat already); that's tracked in #302 .

**Comment by rgossiaux** [comment]
Copying over @asdfryan 's comment from the duplicate:

>We tried this in the past, and did not work. But I know that the League of Legends client does support this somehow. We should take a look at this again.

**Comment by rawxfish** [comment]
I couldn't figure out how to get web links to join Discord voice channels. Discord invites to voice channels when clicked in-client or while in the website will join, but doesn't work when clicked from an outside site. 

The closest solution is to link in cardboard to the text channels, which have a link to join voice. Two clicks, but better than nothing I guess.
